stories/components/icon.stories.js

```diff
@@ -183,3 +183,13 @@ Emergency.args = {
   icon: 'emergency',
   size: 'large',
 };
+
+export const Check = Template.bind({});
+
+Check.args = {
+  icon: 'check',
+  color: 'var(--color-blue-jeans)',
+  circle: true,
+  size: 'large',
+  classes: 'icon--has-background',
```

Comment:
Kann man das nicht eigentlich auch eher über eine Property on off schalten?

Reply:
Wie oben, die Klasse sollte eher nur in der Komponente und im SCSS vorhanden sein.